tests/Integration/Behaviour/Features/Scenario/Carrier/Ranges/carrier_ranges.feature

```diff
@@ -73,39 +76,43 @@ Feature: Carrier ranges
       | zone1   | 300        | 400      | 40          |
       | zone2   | 0          | 100      | 15          |
       | zone2   | 100        | 200      | 25          |
-    Then carrier edit should throw an error with error code "INVALID_RANGES_OVERLAPPING"
+    Then carrier should throw an error with error code "INVALID_RANGES_OVERLAPPING"
 
   Scenario: Get ranges for not all shops
     When I create carrier "carrier1" with specified properties:
       | name             | Carrier 1                          |
       | shippingMethod   | weight                             |
+      | zones            | zone1                              |
```

Comment:
Can you please fix the assertion below, it should use the reference `zone1` not the hard-coded ID (even if an error is triggered before we should remain consistent)